hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -473,6 +505,53 @@ public void close() throws IOException {
     ecChunkBufferCache.release();
   }
 
+  private void addPadding() {
+    final long lastStripeSize =
+        currentBlockGroupLen % (numDataBlks * ecChunkSize);
+
+    final long parityCellSize =
+        lastStripeSize < ecChunkSize ? lastStripeSize : ecChunkSize;
+    ByteBuffer[] buffers = ecChunkBufferCache.getDataBuffers();
+
+    for (int i = 0; i < numDataBlks; i++) {
```

Comment:
I think we should never need to pad the first data buffer. It is the one which drives the size of all others, so we can probably make this look start at 1 rather than zero?

Reply:
Yes, done.